pkg/gardenlet/controller/seed/seed/reconciler_delete.go

```diff
@@ -199,7 +199,7 @@ func (r *Reconciler) runDeleteSeedFlow(
 		autoscaler           = clusterautoscaler.NewBootstrapper(seedClient, r.GardenNamespace)
 		kubeAPIServerIngress = kubeapiserverexposure.NewIngress(seedClient, r.GardenNamespace, kubeapiserverexposure.IngressValues{})
 		kubeAPIServerService = kubeapiserverexposure.NewInternalNameService(seedClient, r.GardenNamespace)
-		kubeStateMetrics     = kubestatemetrics.New(seedClient, r.GardenNamespace, nil, kubestatemetrics.Values{ClusterType: component.ClusterTypeSeed})
+		kubeStateMetrics     = kubestatemetrics.New(seedClient, r.GardenNamespace, nil, kubestatemetrics.Values{ClusterType: component.ClusterTypeSeed}, false)
```

Comment:
What is the impact of defaulting isWorkerless to false?

Reply:
isWorkerless is only relevant for shoots. For seed, setting it to false doesn't affect anything because nothing depends on this field in seed `kube-state-metrics` deployment.